ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-server-groups.j2

```diff
@@ -1,11 +1,10 @@
 {# eos - AAA Group Servers#}
-{% if aaa_server_groups is defined and aaa_server_groups is not none %}
+{% if aaa_server_groups is arista.avd.defined %}
 !
-{%     for aaa_server_group in aaa_server_groups %}
+{%    for aaa_server_group in aaa_server_groups %}
 aaa group server {{ aaa_server_group['type'] }} {{ aaa_server_group['name'] }}
```

Comment:
missing checks for variable `aaa_server_group.type` and `aaa_server_group.name`
Also refactor to use `.` instead of `[ ]`